src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/ReportLatestTests.kt

```diff
@@ -0,0 +1,51 @@
+package org.vaccineimpact.orderlyweb.tests.integration_tests.tests.web
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.models.Scope
+import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
+import org.vaccineimpact.orderlyweb.tests.integration_tests.tests.IntegrationTest
+import org.vaccineimpact.orderlyweb.test_helpers.insertReportWithPublishedAndUnpublishedVersions
+import org.vaccineimpact.orderlyweb.tests.integration_tests.WebPermissionChecker
+
+class ReportLatestTests : IntegrationTest()
+{
+    @Test
+    fun `reviewer can get latest unpublished version`()
+    {
+        val reportName = "test"
+        val publishedVersionId = "20211001-013233-72d597e9"
+        val unpublishedVersionId = "20211002-013233-72d597e9"
```

Comment:
These are redundant as I use them by default in the helper, but I decided to put them here too for readability.

Reply:
I'd be inclined to take out the defaults in the helper in that case.